docker-compose.yaml

```diff
@@ -47,45 +48,34 @@ services:
       HF_TOKEN: ${HF_TOKEN}
     profiles:
       - llamacpp-cpu
-
-  #-----------------------------------
-  #---- Ollama services --------------
-  #-----------------------------------
+    networks:
+      - ollama-net
 
   # Traefik reverse proxy for the Ollama service
-  # This will route requests to the Ollama service based on the profile.
-  ollama:
+  traefik:
     image: traefik:v2.10
     ports:
       - "8081:8080"
+      - "11434:11434"  # Exposing port 11434 for Ollama
     command:
-      - "--providers.file.filename=/etc/router.yml"
-      - "--log.level=ERROR"
-      - "--api.insecure=true"
       - "--providers.docker=true"
-      - "--providers.docker.exposedbydefault=false"
       - "--entrypoints.web.address=:11434"
+      - "--api.insecure=true"
+      - "--log.level=ERROR"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock:ro
-      - ./.docker/router.yml:/etc/router.yml:ro
-    extra_hosts:
-      - "host.docker.internal:host-gateway"
-    profiles:
-      - ""
-      - ollama-cpu
-      - ollama-cuda
-      - ollama-api
+    networks:
+      - ollama-net
 
-  # Ollama service for the CPU mode
-  ollama-cpu:
+  # Ollama service
+  ollama:
```

Comment:
Please go back to `ollama-cpu`. Traeffik will route each request to these children service